app/templates/partials/answers/date.html

```diff
@@ -1,27 +1,48 @@
 {% set subfields %}
- <div class="fieldgroup fieldgroup--date" data-qa="widget-date">
+ <div class="fieldgroup fieldgroup--inline fieldgroup--date" data-qa="widget-date">
   <div class="fieldgroup__fields">
     {% if form.fields[answer['id']]['day'] %}
-    <div class="field field--input field--day">
-      <label class="label mercury" for="{{answer['id']}}-day" id="label-{{answer['id']}}-day" data-qa="label-day">{{ _("Day") }}</label>
-      {% set input = form.fields[answer['id']]['day'] %}
-      {% set input_placeholder = "DD" %}
+    <div class="field field--input">
+      <label class="label u-fs-r" for="{{answer['id']}}-day" id="label-{{answer['id']}}-day" data-qa="label-day">{{ _("Day") }}</label>
+      {% set field = form.fields[answer['id']]['day'] %}
+      {% set input = {
+        'id': field.id,
+        'name': field.name,
+        'classes': 'input--w-2',
+        'type': 'number',
+        'data': field._value()
+      } %}
       {% include 'partials/forms/input.html' %}
     </div>
     {% endif %}
     {% if form.fields[answer['id']]['month'] %}
-    <div class="field field--select field--month">
-      <label class="label mercury" for="{{answer['id']}}-month" id="label-{{answer['id']}}-month" data-qa="label-month">{{ _("Month") }}</label>
-      {% set select = form.fields[answer['id']]['month'] %}
-
+    <div class="field field--select">
+      <label class="label u-fs-r" for="{{answer['id']}}-month" id="label-{{answer['id']}}-month" data-qa="label-month">{{ _("Month") }}</label>
+      {% set field = form.fields[answer['id']]['month'] %}
```

Comment:
Existing but while we're here.
I think it would be better to set the field before the `if` then reuse the variable?
So:
``` Jinja
{% set month_field = form.fields[answer['id']]['month'] %}
{% if month_field %}
	{% set select = {
		'id': month_field.id,
		'name': month_field.id,
		'classes': 'input--w-auto',
		'choices': month_field.choices,
		'data': month_field.data,
		'flags': {
			'required': answer.mandatory
		}
	} %}

	<div class="field field--select">
		<label class="label u-fs-r" for="{{answer['id']}}-month" id="label-{{answer['id']}}-month" data-qa="label-month">{{ _("Month") }}</label>
		{% include 'partials/forms/select.html' %}
	</div>
```
Same concept for day and year also.

Reply:
Done 👍